index.html

```diff
@@ -496,83 +504,62 @@ <h1>Terminology</h1>
                 including communication metadata of WoT Binding
                 Templates.</dd>
             <dt>
-                <dfn>Thing</dfn>
+                <dfn>Thing</dfn> or <dfn>Web Thing</dfn>
             </dt>
             <dd>An abstraction of a physical or a virtual entity
                 whose metadata and interfaces are described by a WoT
                 Thing Description, whereas a virtual entity is the
                 composition of one or more Things.</dd>
-            <dt>
-                <dfn data-lt="WoT Thing Description">Thing
-                    Description</dfn>
-            </dt>
-            <dd>Structured data describing a Thing. A TD includes
```

Comment:
Perhaps the current definition is not ideal and should be improved, however I think we should keep "Thing Description" in the Terminology section.